src/App.jsx

```diff
@@ -14,70 +15,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [food, setFood] = useState('Jam');
```

Comment:
Pay attention to the task description
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/613f8ba0-77f2-4f1d-947e-bb5e0c4c261f)